packages/common/src/sortPeers.ts

```diff
@@ -14,17 +13,19 @@ This is the very simple algorithm for evaluating the most wanted peers.
 export const filterAndSortPeers = (
   peersAddresses: string[],
   stats: NetworkStats[],
-  localPeerAddress?: string
+  localPeerAddress?: string,
+  includeLocalPeerAddress: boolean = true
 ): string[] => {
   peersAddresses = filterValidAddresses(peersAddresses)
+  const currentlyConnected = [...stats].filter(peer => peer.connectionTime === 0)
```

Comment:
When a peer connects we set the `connectionTime` to zero so peers with that value present should be currently connected (or connected the last time we were online).  This means for invitation links the peers are more likely to be online and when we go offline, if we went offline recently, we're more likely to try connecting to online peers (this becomes less likely the longer a user is offline but that's true for connection time filtering, too).

Reply:
Should we be saving who we were last connected to, whenever we shut down, or periodically?

Reply:
That is something I also considered, you can give libp2p a datastore so it can persist the peer data it had on shutdown.